assets/js/phoenix/longpoll.js

```diff
@@ -123,7 +123,7 @@ export default class LongPoll {
 
   batchSend(messages){
     this.awaitingBatchAck = true
-    this.ajax("POST", JSON.stringify(messages), () => this.onerror("timeout"), resp => {
+    this.ajax("POST", "application/ndjson", messages.join("\n"), () => this.onerror("timeout"), resp => {
```

Comment:
According to the spec this should be [`application/x-ndjson`](https://github.com/ndjson/ndjson-spec#33-mediatype-and-file-extensions)

Reply:
Ahhh, you already fixed it 🙈